### ISSUE TITLE
Suggestion: update readme.md

### ISSUE THREAD

**Issue by yahwang** [jtpio/jupyterlab-system-monitor#8]
I used this way to set memory limit and visualize usage in jupyterlab  with reference to nbresuse.

How about updating readme.md for other people?

1. create jupyter_notebook_config.py ( if not using custom config )

```
jupter lab --generate-config
```
2. modify jupyter_notebook_config.py

```
#-----------------------------------------------------------
# NotebookApp(JupyterApp) configuration
#----------------------------------------------------------

c.NotebookApp.ResourceUseDisplay.mem_limit= (Size Of GB) *1024*1024*1024
```

![screenshot](https://user-images.githubusercontent.com/12381530/71590861-67230780-2b6d-11ea-957b-ed4f9417d16a.png)

**Comment by jtpio** [comment]
Thanks @yahwang for the suggestion, this is a good idea!

The readme could also mention the command line argument: `--ResourceUseDisplay.mem_limit= (Size Of GB) *1024*1024*1024`

Would you like to open a PR?

**Comment by yahwang** [comment]
I opened PR #10.

**Comment by jtpio** [comment]
Fixed by #10.
